src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/EditNftsPage/EditNftsSection.tsx

```diff
@@ -38,7 +43,10 @@ export function EditNftsSection() {
         {rewardTiers && rewardTiers.length > 0 && (
           <RewardsList
             value={rewardTiers}
-            onChange={setRewardTiers}
+            onChange={newRewardTiers => {
+              deriveAndSetEditedIds(newRewardTiers)
+              setRewardTiers(newRewardTiers)
```

Comment:
@johnnyd-eth i dont thinkn these should be separate calls. They should happen in one call.

Reply:
Yep good call, changed it up.